db-engine/src/main/java/com/splicemachine/db/impl/sql/compile/WindowResultSetNode.java

```diff
@@ -962,7 +957,10 @@ public ResultSetNode optimize(DataDictionary dataDictionary,
      */
     public void generate(ActivationClassBuilder acb, MethodBuilder mb) throws StandardException {
 
-		/* Get the next ResultSet#, so we can number this ResultSetNode, its
+
+        dataSetProcessorType= CompilerContext.DataSetProcessorType.FORCED_SPARK;
+        acb.setDataSetProcessorType(dataSetProcessorType);
```

Comment:
this change went out of my scope when i change the profile of my intelliJ, thank for catching the issue